FHAC-540 Create / Update AuditLogging Regression Suite tests to include DS Deferred Audit Logging test cases

vdmehta06 opened the pull request:
Regression test suites are created for Standard and Passthrough for DS Deferred Service. It is tested with sample SQL blob message (it is attached with ticket). Basically, for DS Deferred - test suite will verify two entries which will be generated for DS Deferred Request ( one entry on initiating end and other one on responding gateway ) and verify two entries which will be generated for DS Deferred Response ( one entry on initiating end and other one on responding gateway ).

Currently, I have disabled the Verify steps as development changes for DS Deferred Service are not pushed to repository yet.
I have added waiting step (delay) step for Passthrough mode.

vdmehta06 committed:
FHAC-540 Create / Update AuditLogging Regression Suite tests to include DS Deferred Audit Logging test cases

vdmehta06 committed:
FHAC-540 Delay step added for DS Deferred Service

TabassumJafri commented:
PR looks good .. no commets

sailajaa commented on Product/SoapUI_Test/RegressionSuite/Passthru/AuditLogging-Passthrough/DocumentSubmissionDeferredRequestAuditLog.properties:
```diff
@@ -0,0 +1,76 @@
+
+ <!--Destination-->  
+	
+ActiveParticipant_Dest.@UserID=https://localhost:8181/Gateway/DocumentSubmission/2_0/DocumentRepositoryXDR_Service
```
Destination-UserID for DS service is https://localhost:8181/Gateway/DocumentSubmission/2_0/DocumentRepositoryXDR_Service, It will not be same for DS Deferred services. Service names for DS deferred are XDRRequest_Service, XDRResponse_Service. Pls correct the endpoints for both DS deferred Request , Response.

sailajaa commented:
For DS Deferred Response - Responding gateway (B-A)(GwA) , as per design we are not going to populate Patient , SubmissionSet elements, but in PR both elements are there. Pls correct the assertions for DS Deferred Response- Responding Gateway (ACK) message.

vdmehta06 committed:
FHAC-540 addressed PR Comments
- Changed endpoint for g0 and g1 test suite and updated assertions accordingly
- Assertions in ACK for patient and Submission set is removed

vdmehta06 committed:
FHAC-540 - Removed Gateway property directory details

vdmehta06 committed:
FHAC-540 WSDL Preference change in SOAPUI

sailajaa commented:
I reviewed and good with PR updates. Merging the PR.
